### PR TITLE
Demo/120618

### PR DESCRIPTION
**Pull request by LickMyPoppy**
demostuf

### CODE REVIEW

File: Assets/Scripts/Audio/AddThisToEverytScriptThatCallsAudio.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(AudioManagerCombat))]
+[RequireComponent(typeof(AudioManagerMovement))]
+[RequireComponent(typeof(AudioManagerSFX))]
+[RequireComponent(typeof(AudioManagerSkills))]
+
+
+public class AddThisToEverytScriptThatCallsAudio : MonoBehaviour {
```
Comment (jbe1917) [inline Assets/Scripts/Audio/AddThisToEverytScriptThatCallsAudio.cs:11]: EveryScript (Spelling) and maybe different name?

File: Assets/Scripts/Audio/AudioManagerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+	public float combatAudioOffset;
+
+
+	private AudioSource audioSource;
+	private bool walkingSoundsPlaying = false;
+
+
+
+
+	// Use this for initialization
```
Comment (jbe1917) [inline Assets/Scripts/Audio/AudioManagerMovement.cs:30]: default comment

File: Assets/Scripts/Character/CharacterMovement.cs [hunk 3]
```diff
@@ -86,13 +100,13 @@ public class DeathSettings
 	public CharacterController characterController;
 	private Vector3 moveDirection;
     private Statistics statistics;
-	public CharacterAudioController characterAudio;
+	//public CharacterAudioController characterAudio;
```
Comment (jbe1917) [inline Assets/Scripts/Character/CharacterMovement.cs:103]: Why is it commented out? Do we still need it?

File: Assets/Scripts/Character/Statistics.cs [hunk 4]
```diff
@@ -7,6 +7,8 @@ public class Statistics : MonoBehaviour {
 
 	public Slider healthbar;
 
+    public AudioManagerSkills barierAudio;
```
Comment (jbe1917) [inline Assets/Scripts/Character/Statistics.cs:10]: double r

File: Assets/Scripts/Audio/AudioManagerCombat.cs [hunk 5]
```diff
@@ -0,0 +1,172 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioManagerCombat : MonoBehaviour
+{
+
+    public List<AudioClip> Impact;
```
Comment (jbe1917) [inline Assets/Scripts/Audio/AudioManagerCombat.cs:8]: name of all lists should be lower case and maybe have a clearer name like impactSounds or impactClips or something like this

File: Assets/Scripts/Audio/AudioManagerMovement.cs (hunk 2 above)
Comment (LickMyPoppy) [reply Assets/Scripts/Audio/AudioManagerMovement.cs:30]: #ArtProgramming

File: Assets/Scripts/Audio/AddThisToEverytScriptThatCallsAudio.cs (hunk 1 above)
Comment (LickMyPoppy) [reply Assets/Scripts/Audio/AddThisToEverytScriptThatCallsAudio.cs:11]: #ArtProgramming

File: Assets/Scripts/Character/CharacterMovement.cs (hunk 3 above)
Comment (LickMyPoppy) [reply Assets/Scripts/Character/CharacterMovement.cs:103]: done

File: Assets/Scripts/Character/Statistics.cs (hunk 4 above)
Comment (LickMyPoppy) [reply Assets/Scripts/Character/Statistics.cs:10]: done

File: Assets/Scripts/Audio/AudioManagerCombat.cs (hunk 5 above)
Comment (LickMyPoppy) [reply Assets/Scripts/Audio/AudioManagerCombat.cs:8]: #ArtProgramming

**Pull request closed without merging**